gem/templates/profiles/viewprofile.html

```diff
@@ -65,5 +65,13 @@
                 </form>
             </div>
         </div>
+        {% if user.gem_profile %}
+          <hr>
+          <div class="gender-form">
+              <div class="block">
+                      <h5>{% trans "Your gender is:" %}{{user.gem_profile.gender}}</h5>
```

Comment:
You probably need a template tag? That converts the `m` into `male`, etc. Then you can `{{user.gem_profile.gender|expand_gender}}` or something.